### PR TITLE
Feature/design

### PR DESCRIPTION
**Pull request by brysonian**
This PR includes lauren's branch too. A lot of restructuring, refactoring, and cleaning.

### CODE REVIEW

File: src/Components/ProjectHeader/index.js [hunk 1]
```diff
@@ -8,15 +8,15 @@ import {
...
 } from './style.module.css';
 
-const ProjectHeader = ({artistName, title, materials, link}) => (
-  <section className={container}>
+const ProjectHeader = ({artistName, title, materials, link, children}) => (
+  <div className={container}>
     <h3 className={headerArtist}>{artistName}</h3>
     <h2 className={headerTitle}>{title}</h2>
-    <div className={headerMaterials}>{materials}</div>
+    <div className={headerMaterials} dangerouslySetInnerHTML={{__html: materials}} />
```
Comment (blerchin) [inline src/Components/ProjectHeader/index.js:15]: My favorite function :{ should be fine though cause there's no opportunity for XSS

File: src/Components/Image/index.js [hunk 2]
```diff
@@ -7,47 +7,60 @@ import {
...
-  fullHeight: isFullHeight = false,
-  maxHeight = 'auto'
-}) => {
-  const style = {
-    maxHeight,
-  };
-
-  if (maxHeight !== 'auto' && isFullHeight === false) {
-    style.width = 'auto';
+class Image extends React.Component {
```
Comment (blerchin) [inline src/Components/Image/index.js]: I think Hirad ended up making a version of this with hooks. Might have been added after you branched off of `artist-pages`?

File: src/Components/IFrame/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+} from './style.module.css';
+
+const IFrame = ({
+  url,
+  title = 'iframeContent',
+  frameBorder = 0,
+}) => {
+  return (
+    <div className={container}>
+      <iframe
```
Comment (blerchin) [inline src/Components/IFrame/index.js:15]: Does this need to set width and height attrs for the iframe? Usually you have to hack those a bit to make it scale responsively

File: src/work/BenLerchin.js [hunk 4]
```diff
@@ -1,92 +1,60 @@
...
-                        alt="A grey field with the text 1200x1200"
-                        caption="This image has a caption."
+            <ProjectCover>
+                <Image
+                    img={Placeholder}
+                    alt=''
+                />
+            </ProjectCover>
+            <ProjectColumns>
+                <Column>
```
Comment (blerchin) [inline src/work/BenLerchin.js:30]: Yay!

File: src/Containers/App/index.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+  render() {
+    return (
+      <Router>
+        <div className="app">
+          <Suspense fallback={<div />}>
+            <Switch>
+              {
+                config.artists.map(({slug, component}) => {
+                  const Comp = lazy(() => import(`../../work/${component}`));
```
Comment (blerchin) [inline src/Containers/App/index.js:37]: Never seen this before. Rad!

**Review by blerchin — COMMENTED** [review]
Saw this come in while waiting for a video to render, so made some notes. Excited to get this all merged in!

File: src/Components/ProjectHeader/index.js (hunk 1 above)
Comment (brysonian) [reply src/Components/ProjectHeader/index.js:15]: haha, it really is the best. But yeah this is a safe situation for it.

File: src/Components/Image/index.js (hunk 2 above)
Comment (brysonian) [reply src/Components/Image/index.js]: Ahh ok, i'll poke around for it and merge it in.

File: src/Components/IFrame/index.js (hunk 3 above)
Comment (brysonian) [reply src/Components/IFrame/index.js:15]: This was taken from the responsive vimeo component lauren had made and i was trusting that, but does need testing.

File: src/Containers/App/index.js (hunk 5 above)
Comment (brysonian) [reply src/Containers/App/index.js:37]: Yeah it is great. We had a single chunk at 500k, not too bad really. But this gives a big one at 155, then a smaller chunk for each artist route that gets loaded as-needed.

File: src/Components/Image/index.js (hunk 2 above)
Comment (brysonian) [reply src/Components/Image/index.js]: Ahh looks like it is in here: #24

**Comment by brysonian** [comment]
Just merged master

**Comment by hsab** [comment]
Chandler this is great! Only thing that seems broken is the bg title on the homepage

**Comment by blerchin** [comment]
sorry, must have clicked the close button by accident 🤔 
once we figure out the title issue this should be good to go

**Review by blerchin — APPROVED** [review]
👍 🦄

**Pull request merged**